org/istio.yaml

```diff
@@ -15,6 +14,7 @@ orgs:
     - rshriram
     - smawson
     members:
+    - geeknoid
```

Comment:
this entry must be sorted alphabetically. I would tell you the line number, but this file changes often, so the line number would change often.

You should move this entry to between 73 and 74 at the current state of the repo.